gips/data/landsat/landsat.py

```diff
@@ -807,6 +807,17 @@ def filter(self, pclouds=100, sensors=None, **kwargs):
                 return False
         return True
 
+    def masks(self, patterns=None):
```

Comment:
Can this method be changed to make a super call to `Data.masks(['acca', 'fmask', 'mask'])`?  It looks like it works so this is only a suggestion.